src/web_app_skeleton/src/app.cr.ecr

```diff
@@ -50,4 +50,13 @@ class App < Lucky::BaseApp
       Lucky::RouteNotFoundHandler.new,
     ]
   end
+
+  def protocol
+    "http"
+  end
+
+  def listen
+    server.bind_tcp(host, port, false)
```

Comment:
I love the idea of showing people they can use the reuse port option. To make it a bit clearer what if we added a keyword arg: `server.bind_tcp(host, port, reuse_port: false)`. Maybe the comment could then be a link to the `bind_tcp` docs in Crystal

Reply:
ah yup. Good call.